### ISSUE TITLE
[Bug][2] Problem with iPhone sign in

### ISSUE THREAD

**Issue by polubis** [polubis/4markdown#74]
**Context**

The authorization method with Google Provider does not work on iPhone 12 and others. It does not work for all browsers. 

**Steps to Reproduce**

1. Sign in
2. You'll see an error

**Definition of Done**

1. The user should be able to sign in on an iPhone device with every supported browser - check [USER_STORY](https://github.com/users/polubis/projects/4/views/8?pane=issue&itemId=58069469)

**Comment by polubis** [comment]
https://github.com/polubis/4markdown/pull/214

**Comment by polubis** [comment]
May be tested here: https://soft-pithivier-093469.netlify.app/

**Comment by KaminskiKrzysztof** [comment]
DOD:
✅ I was finally able to log into my account with iPhone 12 on Google Chrome, Safari, Firefox 
❌ the same issue still occurs on Opera <- I still can't log into my account on this browser

(?) The only thing that concerns me is the link presented while choosing an account to log into the application. Lmk if this is an acceptable result

![Image](https://github.com/polubis/4markdown/assets/159806817/f0b98170-e272-46f9-93b0-4261850deda6)

**Comment by polubis** [comment]
Yes, the link is weird because it's develop environment, on PROD it will be valid one.
